gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckerPlugin.java

```diff
@@ -27,5 +27,5 @@
   public void apply(Project project) {
     project.getExtensions().create("linkageChecker", LinkageCheckerPluginExtension.class);
     project.getTasks().create("linkageCheck", LinkageCheckTask.class);
-  }
+//  }
```

Comment:
Confirmed this syntax error the gradle-plugin module fails the builds

<img width="856" alt="Screen Shot 2020-08-24 at 00 47 28" src="https://user-images.githubusercontent.com/28604/91004949-87438300-e5a3-11ea-8af0-ec4693c97a14.png">